uber/templates/preregistration/preregbase.html

```diff
@@ -95,7 +101,9 @@
                 price: {{ c.BADGE_TYPE_PRICES[badge_type] }},
                 badge_type: '{{ badge_type }}',
                 onClick: function () {
-                    $('input[name="badge_type"]').val('{{ badge_type }}');
+                    if ($('input[name="badge_type"]').val() != '{{ c.STAFF_BADGE }}') {
```

Comment:
I had an idea for that problem actually. If the attendee's current `badge_type` is not one of the `PREREG_BADGE_TYPES` then the template will set the "Attendee" option to whatever the current `badge_type` is.

So, let's say they have a staff badge, then under the hood the "Attendee" button will actually set the `badge_type` to staff. If they click on "Shiny Sponsor" and then click back to "Attendee", they'll maintain their staff badge.

But you can fiddle with that later, if this is a non-MAGFest thing.